src/deprecated.jl

```diff
@@ -0,0 +1,31 @@
+doi(x::Record) = x.location.doi
+function doi(x::RecordDatabase) 
+  dois = []
+     for i in keys(x)
+         push!(dois, location(get(x.records, i, false)).doi)
+     end
+  return dois
+end
+
+@deprecate doi(x) url(x)
+
+function check_doi(x::RecordDatabase, y::Record)
```

Comment:
Just for my understanding: Why would we keep these checks at all? I'd think it makes sense for the extractors, but why the checks?